### DISCUSSION TITLE
Youtube saying "Please use a keyframe frequency of four seconds or less."

### DISCUSSION THREAD

**Discussion by mariia-artamonova** [ant-media/Ant-Media-Server#5952] [FAQs] [answered]
Hello

OK so I watched a popular YouTube video of [how to create a 24/7 YouTube Livestream](
https://www.youtube.com/watch?v=0scjy6Zxzwcwed) where he used Linode to connect to Ant Media Server.

The issue I'm having is that YouTube says I have poor stream health - see message below:

"Please use a keyframe frequency of four seconds or less. Currently, keyframes are not being sent often enough, which can cause buffering. The current keyframe frequency is 6.8 seconds. Note that ingestion errors can cause incorrect GOP (group of pictures) sizes."

How can I configure this in Ant Media whilst connected to Linode?

Please let me know.

Thanks!

**Comment by yashtandon113** [comment, marked as answer]
The keyframe frequency and other parameters depend on the VOD file that you are using to create a playlist.

The recommended upload encoding settings for YouTube streaming can be found at [this link](https://support.google.com/youtube/answer/1722171?hl=en#zippy=).

So in this case, you need to reencode your video with the required parameters.

You can use FFMPEG to change the key frame interval. The below command will re-encode the file with a key frame interval of 2.
 
`ffmpeg -i input.mp4 -qscale 0 -g 2 outputFile.mp4`

You can also use the HANDBRAKE tool to re-encode the video. Please check out [this article](https://help.vidswap.com/hc/en-us/articles/217120566-Encoding-Re-Encoding-Video-and-or-Wiping-Audio-Using-HANDBRAKE).

I hope it helps.

**Reply by sampsoe** [reply]
Thank you very much, I will have a go of this information...
